example/README.md

```diff
@@ -170,37 +170,57 @@ for rebooting the `pxe-1` virtual machine.
 For example, you can [manually download the ESXi installation ISO](https://www.vmware.com/go/download-vspherehypervisor)
 or download a [CentOS 7 Installation ISO](http://mirrors.mit.edu/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1511.iso).
 
-Copy it into the `examples` directory and then you can unpack it in vagrant:
+**NOTE:** Below, we show two methods (A&B) of insuring we have the iso file properly placed to be referenced by our helper script.
 
-`vagrant ssh`:
-
-    sudo mkdir /var/mirrors
-    sudo python ~/src/on-http/data/templates/setup_iso.py /vagrant/VMware-VMvisor-Installer-*.x86_64.iso /var/mirrors --link=/home/vagrant/src
 
+---
+**A.** Copy the iso into the `examples` directory within the RacKHD directory and then you can unpack it in vagrant:
 
 `vagrant ssh`:
+```    
+sudo python ~/src/on-tools/scripts/setup_iso.py /vagrant/VMware-VMvisor-Installer-*.x86_64.iso /opt/monorail/static/http --link=/home/vagrant/src
+```
+
 
-    cd /tmp
-    wget http://mirrors.mit.edu/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1511.iso
-    # 4GB
-    sudo python ~/src/on-http/data/templates/setup_iso.py /tmp/CentOS-7-x86_64*.iso /var/mirrors --link=/home/vagrant/src
+---
+**B.** Below shows downloading an iso file directly onto the server.
 
+**NOTE:** We continue with a CentOS install to complete the instructions.
+
+`vagrant ssh`:
+```
+cd /tmp
+wget http://mirrors.mit.edu/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1511.iso
+# 4GB
+sudo python ~/src/on-tools/scripts/setup_iso.py /tmp/CentOS-7-x86_64*.iso /opt/monorail/static/http --link=/home/vagrant/src
```

Comment:
These steps could be combined. Setup_iso.py can be given a download url instead of a file in the arguments.